airflow/utils/state.py

```diff
@@ -60,6 +60,7 @@ class DagRunState(str, Enum):
     same name in TaskInstanceState.
     """
 
+    QUEUED = 'queued'
```

Comment:
```suggestion
    QUEUED = "queued"
```

nit.. since there one below have double quotes